pyproject.toml

```diff
@@ -19,8 +19,8 @@ classifiers = [
     "Topic :: Scientific/Engineering :: GIS",
     "Typing :: Typed",
 ]
-version = "0.2.1"
-dependencies = ["pydantic>=2.3,<3"]
+version = "0.3.0"
```

Comment:
Is this a breaking change?

Reply:
No, when upgrading you can still use the FeatureCollection of [geojson-pydantic](https://github.com/developmentseed/geojson-pydantic). It is not mandatory to use the model or implement the parameters.

Reply:
Then why bump minor version?